### ISSUE TITLE
After rerender table using Cells flickers and scrolls back to the top

### ISSUE THREAD

**Issue by henrikra** [aksonov/react-native-tableview#192]
Code for reproduction is here: https://github.com/henrikra/react-native-tableview/blob/append-items-bug/example/src/screens/Example9.js

## What I would like to achieve
I would like to achieve that always when I add new items to the end of the list the list will scroll to the bottom. 

## What works 💚 

I can achieve that when using `Item` from this library. 
### Gif
![bug works](https://user-images.githubusercontent.com/12229968/48132505-7c1f4600-e29c-11e8-87ca-4abc9b4dab83.gif)

## What doesn't work 🔴 

When I use exactly the same code except I use `Cell` instead of `Item` it will flash and also start from the top again and also it does not animate to the bottom.
### Gif
![bug does not work](https://user-images.githubusercontent.com/12229968/48132621-e6d08180-e29c-11e8-8bfb-6b6d3d7ed3d1.gif)

## Conclusion

Looks like bug to me 😢 I would really hope this will get fixed because `Item` is too limited for me. I have to style the component. I can help also to resolve this issue if you guys point me to right place to look.

If this is not fixable do you have other suggestions on these kind of React Native iOS libraries?

**Comment by saadqbal** [comment]
Hi @henrikra have you found any good solution for your chat List ? I have tried a number of options but nothing has worked so far. 

Tried RecycleListView, react-native-largelist but none has worked.

**Comment by henrikra** [comment]
Nope. I just ended up using FlatList on iOS which is not ideal :/

**Comment by nyura123** [comment]
@henrikra for styling `Item`s, you could try using https://github.com/aksonov/react-native-tableview/blob/master/README.md#editable-complex-components
